test-infra/ansible/inventory/hosts.yml

```
@@ -0,0 +1,17 @@
+      hosts:
+    db_servers:
+      hosts:
+    switches:
+      hosts:
+    production:
+      hosts:
+    test:
+      hosts:
+        example_name:
+          ansible_host: ip_address 
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)